app/components/HexagonText.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from "react";
+
+export const HexagonText: React.SFC = () =>
+<div>
```

Comment:
please move it inside Incentive.tsx - it's too specific component to make a separate file for it.